experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -156,7 +156,9 @@ function ClimaLandSimulation(
     Y.snow.S_l .= FT(0)
     Y.snow.U .= FT(0)
 
-    set_initial_cache! = CL.make_set_initial_cache(model)
+    # Initialize the surface temperature so the atmosphere can compute radiation
```

Comment:
Let's maybe add a little bit more detail here (so that people can get more context in the future)